infrastructure-readme.md

```diff
@@ -0,0 +1,15 @@
+# Servers
+
+Heritage uses servers in Alberta as their primary: mamirolle.uab cheddar.uab
+
+But the other portals use Toronto: iris.tor gouda.tor jarlsberg.tor
+
+The two production Solr databases are iris and gouda.
+
+
+# Solr
+A Solr GUI exists and can be accessed here, if connected to OpenVPN: http://.c7a.ca:8983/solr/#/cosearch2
+
+
+# Couch
+A GUI for Couch exists, and can be accessed here, if connected to OpenVPN: http://.c7a.ca:5984/_utils/#/database/dmdtask/_all_docs
```

Comment:
You can leave as http://.c7a.ca:5984/_utils/

The rest is specific to a database that is only on Jarlsberg :-) 

Same with http://.c7a.ca:8983/  for Solr, as the rest is a specific database.


There is currently a CouchDB on Jarlsberg used for the Metadatabus and other tools, but not a Solr.  It isn't one of the Patron production servers, but a server for manipulation of data (ingest, metadatabus, OCR manipulation, DMD manipulation, etc).